groups/bsl/bslstl/bslstl_badvariantaccess.t.cpp

```diff
@@ -0,0 +1,279 @@
+// bslstl_badvariantaccess.t.cpp                                      -*-C++-*-
+#include <bslstl_badvariantaccess.h>
+
+#include <bslmf_assert.h>
+#include <bslmf_issame.h>
+
+#include <bsls_asserttest.h>
+#include <bsls_bsltestutil.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+using namespace BloombergLP;
+
+// ============================================================================
+//                             TEST PLAN
+// ----------------------------------------------------------------------------
+//                             Overview
+//                             --------
+// The type under test is 'bsl::bad_variant_access', an exception type whose
+// interface and contract are dictated by the C++ standard.  If the
+// 'std::bad_variant_access' exception is available, we need to check that
+// 'bsl::bad_variant_access' is a typedef to the standard's exception type.  If
+// the 'std::bad_variant_access' exception isn't available, we need to check
+// that 'bsl::bad_variant_access' satisfies the interface and contract of
+// 'std::bad_variant_access'.
+//
+// ----------------------------------------------------------------------------
+// CREATORS
+// [ 3] bad_variant_access();
+//
+// ACCESSORS
+// [ 3] const char *what() const;
+//
+// typedef:
+// [ 4] typedef bsl::bad_variant_access
```

Comment:
Since this is now a *using-declaration* alias, it shouldn't be indicated as a `typedef` anymore. Although tbh, I'm not sure what the correct BDE style is for this.

Reply:
I wasn't able to find precedent for this after doing a cursory look over bslstl. So, here's my suggestion: we use the heading `TYPES` (to match what we do in class definitions) and drop the word `typedef`. I don't really have any better ideas.
```suggestion
// TYPES
// [ 4] bsl::bad_variant_access
```
Please remember to make the corresponding change on line 138.